docs/advanced/UsageWithTypescript.md

```diff
@@ -56,47 +56,45 @@ Chat Action Constants & Shape:
 // src/store/chat/types.ts
 
 export enum ChatActions {
-  SendMessage = 'SEND_MESSAGE'
+  SendMessage = 'SEND_MESSAGE',
+  DeleteMessage = 'DELETE_MESSAGE'
 }
 
 interface SendMessageAction {
   type: ChatActions.SendMessage
   payload: Message
 }
 
-export type ChatActionTypes = SendMessageAction
-```
-
-Note that you can use TypeScript's Union Type to express multiple actions like so:
+interface DeleteMessageAction {
+  type: ChatActions.DeleteMessage
+  timestamp: number
```

Comment:
Shouldn't we encourage using [FSA](https://github.com/redux-utilities/flux-standard-action)? If so, `timestamp` should be `meta: { timestamp }` instead.

Reply:
If I am not mistaken the flux library is what suggested the `payload` and `meta` format for actions. Redux actions do not require a meta field, only the type field is required. I would like to keep actions and its nomenclature consistent with the rest of the documentation. Another point to take into consideration is that many people who learn Redux aren't coming from the flux library anymore.

Reply:
Not quite.  `payload` and `meta` are from the ["Flux Standard Actions" community standard](https://github.com/redux-utilities/flux-standard-action), which was developed by Andrew Clark.  While Redux doesn't require FSA structure, a very large percentage of Redux apps use an FSA-based approach.

Reply:
Thanks for clarifying, I'll take that as changing the actions in this example to follow FSA.